test/integration/features/cmd-openshift.feature

```diff
@@ -0,0 +1,115 @@
+@cmd-openshift
+Feature: Openshift commands
+Commands "minishift openshift [sub-command]" are used for interaction with Openshift
+cluster in VM provided by Minishift.
+
+  Scenario: Trying service command when Minishift is not running
+     Given Minishift has state "Does Not Exist"
+      When executing "minishift openshift service list" succeeds
+      Then stdout should contain
+      """
+      Running this command requires an existing 'minishift' VM, but no VM is defined.
+      """
+
+  Scenario: Minishift start
+  Minishift must be started in order to interact with OpenShift via "minishift openshift" command
+      When executing "minishift start" succeeds
+
+  Scenario: Service list sub-command
+     Given Minishift has state "Running"
+      When executing "minishift openshift service list" succeeds
+      Then stdout should contain "docker-registry"
+       And stdout should contain "kubernetes"
+       And stdout should contain "router"
+
+  Scenario: Restarting the OpenShift cluster
+     Given stdout of command "minishift ssh -- docker inspect --format={{.State.FinishedAt}} origin" is equal to "0001-01-01T00:00:00Z"
+      When executing "minishift openshift restart" succeeds
+      Then stdout should contain "Restarting OpenShift"
+       And stdout of command "minishift ssh -- docker inspect --format={{.State.FinishedAt}} origin" is not equal to "0001-01-01T00:00:00Z"
```

Comment:
so is the '.State.FinishedAt' field then referencing the time when this container had finished the last time?